client/src/modules/list/components/Items/InputBar/InputBar.scss

```diff
@@ -11,6 +11,7 @@
     transform: translateY(-50%);
     width: auto;
     text-transform: uppercase;
+    min-height: 36px;
```

Comment:
why not using constant?

Reply:
how would I name this constant and where to place them?
If I should use constant here, should I create constant for every place where I have 37px, 38px, 39px??

Reply:
I thought it was margin :) now I see it is height. Ignore it :)